roles/nfs-client-provisioner/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+# Vars needed to install nfs-client-provisioner
+k8s_nfs_server: "127.0.0.1"
+k8s_nfs_export_path: "/export/shared"
+k8s_nfs_default_sc: "true"
+k8s_nfs_client_repo_name: "stable"
+k8s_nfs_client_helm_repo: "https://charts.helm.sh/stable"
```

Comment:
This can't be stable long term.

Reply:
Given that this will always be enabled, should we make `k8s_nfs_export_path` something more uniquely named?  Like `/export/deepops_nfs` or the like?

Reply:
That's a great idea. Is /exports the standard location to export as an nfs path?

Reply:
Using `/exports` is arbitrary, but a common practice.  I think keeping that as the top level directory is as good as anything, and helps make clear that these/this is an "exported" filesystem.